engine/src/main/java/org/hibernate/validator/internal/engine/messageinterpolation/ParameterTermResolver.java

```diff
@@ -52,4 +48,41 @@ private Object getVariable(Context context, String parameter) {
 	private String removeCurlyBraces(String parameter) {
 		return parameter.substring( 1, parameter.length() - 1 );
 	}
+
+	private String resolveExpressionByNotNullVariable(Object variable) {
```

Comment:
I would just rename that method to `resolveExpression`. No need to make it extra verbose.

Reply:
fixed